tests/unit/test_rg_matrix.cpp

```diff
@@ -737,3 +737,173 @@ TEST_F(RGMatrixTest, RGMatrix_fuzzy) {
 	free(I);
 	free(J);
 }
+
+
+TEST_F(RGMatrixTest, RGMatrix_export_no_changes) {
```

Comment:
Add a comment explaining the purpose of this test, and in general what's going on here, (introduction)